SQL/0000-00-03-ConfigTables.sql

```diff
@@ -29,15 +32,16 @@ CREATE TABLE `ConfigSettings` (
 CREATE TABLE `Config` (
   `ID` int(11) NOT NULL AUTO_INCREMENT,
   `ConfigID` int(11) NOT NULL,
-  `Value` text,
+  `Value` varchar(767),
```

Comment:
Since we do not set a limit to the size of text that we can enter, i think keeping TEXT is safer !! (especially that non-strict databases do not throw an error on save if data is truncated !)

this will also reduce the amount of changes in this PR

Reply:
I agree with @ridz1208 .. there's no reason for any changes other than the check constraint (and patch) to accomplish what the title/description says the problem is.